src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,127 @@
+
+        try {
+            runCycle(today, crewGroup);
+        } catch (Exception e) {
+            outputView.printError(e.getMessage());
+        }
+    }
+
+    private void runCycle(LocalDateTime today, CrewGroup crewGroup) {
+        while (true) {
+            String option = inputView.insertMenuOption(today);
+            if (isExit(option)) {
+                return;
+            }
+            operateMenuOption(option, crewGroup, today);
+        }
+    }
+
+    private void operateMenuOption(String option, CrewGroup crewGroup, LocalDateTime today) {
+        MenuOption menuOption = getMenuOption(option);
+        if (menuOption.equals(CHECK_ATTENDANCE)) {
+            attendanceCheck(crewGroup, today);
+        }
+        if (menuOption.equals(CHECK_ATTENDANCE)) {
+            changeAttendance(crewGroup, today);
+        }
+        if (menuOption.equals(SHOW_CREW_ATTENDANCES)) {
+            showCrewAttendance(crewGroup);
+        }
+        if (menuOption.equals(SHOW_ALERT_CREW)) {
+            showAlertCrews(crewGroup);
+        }
+    }
+
+    private void attendanceCheck(CrewGroup crewGroup, LocalDateTime today) {
+        DateValidator.validateAttendanceCheckDate(today);
+
+        String rawName = inputView.insertNickname();
+        Crew crew = crewGroup.searchCrew(rawName);
+        if (crew.isAlreadyChecked(today)) {
+            outputView.printGuide();
+            return;
+        }
+
+        String rawTime = inputView.insertTime();
+        Time time = new Time(rawTime);
+        Attendance attendance = crew.addAttendance(
+                LocalDateTime.of(today.getYear(), today.getMonth(), today.getDayOfMonth(),
+                        time.getHour(), time.getMinute()));
+
+        outputView.printAttendanceLog(AttendanceLogDTO.from(attendance));
+    }
+
+    private void changeAttendance(CrewGroup crewGroup, LocalDateTime today) {
+        String rawName = inputView.insertChangeDateNickname();
+        Crew crew = crewGroup.searchCrew(rawName);
+
+        int date = inputView.insertChangeDate();
+        DateValidator.validateAttendanceChangeDate(date, today);
+
+        String rawTime = inputView.insertChangeTime();
+        Time time = new Time(rawTime);
+
+        Attendance originalAttendanceCopy = new Attendance(crew.getSpecificAttendance(date).getDate());
+        Attendance changedAttendance = crew.changeAttendance(date, time);
+
+        outputView.printChangeLog(ChangeAttendanceLogDTO.from(originalAttendanceCopy, changedAttendance));
+    }
+
+    private void showCrewAttendance(CrewGroup crewGroup) {
+        String rawName = inputView.insertNickname();
+        Crew crew = crewGroup.searchCrew(rawName);
+
+        CrewAttendancesDTO crewAttendancesDTO = CrewAttendancesDTO.from(crew);
+
+        outputView.printAttendancesLog(crewAttendancesDTO);
+    }
+
+    private void showAlertCrews(CrewGroup crewGroup) {
+        List<AlertCrewDTO> alertCrewDTDs = crewGroup.getAllAttendanceAlertLevel()
```

Comment:
```suggestion
        List<AlertCrewDTO> alertCrewDTOs = crewGroup.getAllAttendanceAlertLevel()
```
DTO 클래스명도 보통은 카멜 케이스(`AlertCrewDto`)로 작성합니다.

Reply:
넵! 감사합니다 전부 카멜케이스로 변경하였습니다.